src/main/java/com/salesforce/mirus/KafkaMonitor.java

```diff
@@ -89,6 +96,51 @@
         config.getEnablePartitionMatching()
             ? SourcePartitionValidator.MatchingStrategy.PARTITION
             : SourcePartitionValidator.MatchingStrategy.TOPIC;
+    this.routers = this.validateTransformations(config.transformations());
+    this.topicNamePrefix = config.getDestinationTopicNamePrefix();
+    this.topicNameSuffix = config.getDestinationTopicNameSuffix();
+  }
+
+  private List<Transformation<SourceRecord>> validateTransformations(
+      List<Transformation<SourceRecord>> transformations) {
+    List<Transformation<SourceRecord>> regexRouters = new ArrayList<>();
+    for (Transformation<SourceRecord> transform : transformations) {
+      String transformName = transform.getClass().getName();
+      if (transform instanceof RegexRouter) {
+        regexRouters.add(transform);
+        // Slightly awkward check to see if any other routing transforms are configured
+      } else if (transform.getClass().getName().contains("Router")) {
+        logger.error(
```

Comment:
I think as long as we document this behaviour clearly it's not really an error. I think we should make it a `warn`, and say something like "Destination topic checking is not supported with router transform %s enabled. Please ensure all possible destination topics exist".  Also, probably should disable all destination topic checking in that case as might end up checking for topics that are not intended to exist?